pkg/vmprovider/providers/vsphere2/vmlifecycle/bootstrap.go

```diff
@@ -161,9 +177,14 @@ func getBootstrapArgs(
 			return nil, err
 		}
 
-		// GOSC will use these for its global config.
-		bootstrapArgs.DNSServers = nameservers
-		bootstrapArgs.SearchSuffixes = searchSuffixes
+		if len(bootstrapArgs.DNSServers) == 0 {
+			// GOSC will this for its global config.
+			bootstrapArgs.DNSServers = nameservers
+		}
+		if !isGOSC && len(bootstrapArgs.SearchSuffixes) == 0 {
```

Comment:
Do we *really* care if we set `bootstrapArgs.SearchSuffixes` to `searchSuffixes` if GOSC is used? I mean, they will be ignored if it is. What does this achieve other than saving the memory used by a slice?